examples/mysql-kopia-repository-server/README.md

```diff
@@ -0,0 +1,423 @@
+   # optional: if set to true, do not verify SSL cert.
+   # Default, when omitted, is false
+   #skipSSLVerify: false
+   # required: if type is `file-store`
+   # optional, otherwise
+   #claimName: store-pvc
+```
+
+- Apply Secrets
+```bash
+$ kubectl create secret tls repository-server-tls-cert --cert=certificate.pem --key=key.pem -n kanister
+
+kubectl create secret generic repository-server-user-access -n kanister --from-literal=localhost=<suitable_password_for_repository_server_user>
+
+kubectl create secret generic repository-admin-user -n kanister --from-literal=username=admin@mysql --from-literal=password=<suitable_password_for_repository_server_admin>
+
+kubectl create secret generic repo-pass -n kanister --from-literal=repo-password=<repository_password_set_while_creating_kopia_repository>
+
+kubectl create -f s3_location_creds.yaml -n kanister
+
+kubectl create -f s3_location.yaml -n kanister
+```
+
+- Apply Repository Server CRD
+```bash
+$ kubectl create -f pkg/customresource/repositoryserver.yaml -n kanister
+```
+
+- Create Repository Server CR
+```bash
+vi repo-server-cr.yaml 
+```
+```
+apiVersion: cr.kanister.io/v1alpha1
+kind: RepositoryServer
+metadata:
+  labels:
+    app.kubernetes.io/name: repositoryserver
+    app.kubernetes.io/instance: repositoryserver-sample
+    app.kubernetes.io/part-of: kanister
+    app.kuberentes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: kanister
+  name: kopia-repo-server-1
+  namespace: kanister
+spec:
+  storage:
+    secretRef:
+      name: s3-location
+      namespace: kanister
+    credentialSecretRef:
+      name: s3-creds
+      namespace: kanister
+  repository:
+    rootPath: /repo-controller/
+    passwordSecretRef:
+      name: repo-pass
+      namespace: kanister
+    username: kanisterAdmin
+    hostname: mysql.app
+  server:
+    adminSecretRef:
+      name: repository-admin-user
+      namespace: kanister
+    tlsSecretRef:
+      name: repository-server-tls-cert
+      namespace: kanister
+    userAccess:
+      userAccessSecretRef:
+        name: repository-server-user-access
+        namespace: kanister
+      username: kanisteruser
+```
+```bash
+$ kubectl create -f repo-server-cr.yaml -n kanister
+```
+**NOTE:**
+
+Make Sure the Repository Server is in ServerReady State before creating actionsets.
+You could check the status of the Repository Server CR by running following command
+```bash
+$ kubectl get repositoryservers.cr.kanister.io kopia-repo-server-1 -n kanister -o yaml
+```
+
+**NOTE:**
+
+The above command will configure a kopia repository server, which manages artifacts resulting from Kanister
+data operations such as backup should go. This is stored as a `repositoryservers.cr.kanister.io`
+*CustomResource (CR)* which is then referenced in Kanister ActionSets. Every ActionSet
+requires a Repository Server reference to complete the action. This CR (`repositoryservers.cr.kanister.io`)
+can be shared between Kanister-enabled application instances.
```

Comment:
> This CR (`repositoryservers.cr.kanister.io`)
> can be shared between Kanister-enabled application instances.

What do you mean by this?

Reply:
It means like we can use repository server with all the applications which are supported by Kanister, But I get your point.